contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -137,8 +137,40 @@ var ContentNodeModel = BaseModel.extend({
 		assessment_items:[],
 		metadata: {"resource_size" : 0, "resource_count" : 0},
 		created: new Date(),
-		ancestors: []
-    }
+		ancestors: [],
+		extra_fields: {}
+    },
+    initialize: function () {
```

Comment:
Set extra_fields as a JSON object on initialization to avoid having to parse it each time it gets used